src/components/Reservations/Reservations.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable max-len */
+import React, { useEffect, useState } from 'react';
+import Button from 'react-bootstrap/Button';
+import { useDispatch, useSelector } from 'react-redux';
+import { useSearchParams } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Spinner from 'react-bootstrap/Spinner';
+import Pagination from 'react-bootstrap/Pagination';
+import { fechReservations } from '../../redux/reservations/reservations';
+import { deleteReservations, clearDeleteReservationState, clearFetchReservationState } from '../../redux/reservations/deleteReservation';
```

Comment:
Importing `clearFetchReservationState` may not be necessary here. Please kindly remove it. 

```suggestion
import { deleteReservations, clearDeleteReservationState } from '../../redux/reservations/deleteReservation';
```